internal/cmd/sync.go

```diff
@@ -95,7 +98,7 @@ func makeRunSync(init *bool) func(cmd *cobra.Command, args []string) error {
 		if rsyncErr != nil {
 			flog.Info("Unable to determine remote rsync version.  Proceeding cautiously.")
 		} else if localVersion != remoteVersion {
-			flog.Fatal("rsync protocol mismatch: local = %v, remote = %v", localVersion, rsyncErr)
+			return xerrors.Errorf("rsync protocol mismatch: local = %s, remote = %s", localVersion, rsyncErr)
```

Comment:
might want to remove rsyncErr here as we know it will be nil. I suspect it might have been a typo for remoteVersion?